backend/src/core/middlewares/api-key.middleware.ts

```diff
@@ -54,10 +55,33 @@ export const InitApiKeyMiddleware = (
     const userId = req.session?.user?.id
     const apiKey = await credentialService.generateApiKey(
       +userId,
-      req.body.label
+      req.body.label,
+      req.body.notification_addresses
     )
     return res.status(201).json(convertApiKeyModelToResponse(apiKey))
   }
+  const updateApiKey = async (
```

Comment:
**[question]** so right now, this only updates notification addresses, but in the future, this endpoint will be used to update the other stuff too right (permissions? anything else?)

Reply:
yes